gatsby/pagination/create-courses-pages.js

```diff
@@ -1,3 +1,4 @@
+/* @deprecated -- courses page fetches all courses at once  */
```

Comment:
Our new pagination strategy doesn't create a new page for each page of courses. They technically all exist within the same page, but hide/appear when you move to the next page. I feel like this could be something we might want eventually so I didn't want to fully remove the logic.